LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Atlassian Pty Ltd
```

Comment:
Put Atlassian here since we probably want to move this lib into the Atlassian org at some point.

Reply:
Already created an issue for it a while back. Thinking of picking it up after I wrap up the work for the github install flow stream.